docs/custom_promise_types/site_up.py

```diff
@@ -0,0 +1,45 @@
+import re
+import ssl
+import urllib.request
+import urllib.error
+from cfengine import PromiseModule, ValidationError, Result
+
+
+class SiteUpPromiseTypeModule(PromiseModule):
+    def __init__(self):
+        super().__init__("site_up_promise_module", "0.0.1")
+
+    def validate_promise(self, promiser, attributes):
+        if self.is_url_valid(promiser) == False :
+            raise ValidationError(f"URL '{promiser}' is invalid")
+
+
+    def evaluate_promise(self, url, attributes):
```

Comment:
```suggestion
            raise ValidationError(f"URL '{promiser}' is invalid")

    def evaluate_promise(self, url, attributes):
```

Nitpicking, but shouldn't be 2 lines between here. I think [black](https://github.com/psf/black) would remove this.